packages/babel-runtime/scripts/build-dist.js

```diff
@@ -3,8 +3,8 @@
 const outputFile = require("output-file-sync");
 const coreDefinitions = require("@babel/plugin-transform-runtime").definitions;
 const helpers = require("@babel/helpers");
-const babel = require("../../babel-core");
-const t = require("../../babel-types");
+const babel = require("../../@babel/core");
```

Comment:
Since we're using them as normal devDependencies, you'll want to remove the `../../` from these to make them work.

Reply:
Thank you, @loganfsmyth